packages/snaps-types/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import { StreamProvider } from '@metamask/providers';
+
+/**
+ * The type of `window.ethereum`.
+ */
+export type Ethereum = StreamProvider;
+
+// Exported again for convenience.
+export type { Json, JsonRpcRequest } from '@metamask/utils';
+export type {
```

Comment:
Is there a reason we are not exporting `SnapFunctionExports` and `SnapExports`?

Reply:
I don't think external consumers need those? Is there any good use case for them?

Reply:
Most likely not - Just curious if you had thought about it. Since this is breaking its fine if we remove it, just add it to the PR description